pulser-core/pulser/devices/_device_datacls.py

```diff
@@ -621,6 +621,10 @@ def _optional_parameters(self) -> tuple[str, ...]:
     def change_rydberg_level(self, ryd_lvl: int) -> None:
         """Changes the Rydberg level used in the Device.
 
+        Find the $C_6$ coefficient matching the Rydberg level on
+        `this page <https://github.com/pasqal-io/Pulser/blob/develop/
+        pulser-core/pulser/devices/interaction_coefficients/C6_coeffs.json>`
+
```

Comment:
Hum, this didn't work out unfortunately...
<img width="754" alt="image" src="https://user-images.githubusercontent.com/29920212/212295007-2afa5bc8-672e-4497-b8d0-6792e0d3af54.png">